site/src/utils/uri.ts

```diff
@@ -0,0 +1,32 @@
+export const formatURI = (uri: string) => {
```

Comment:
This function's name was fine when it was local to the AppStatuses page, but it doesn't seem to be descriptive enough to be a project-wide helper function. Maybe something along the lines of `truncateURIForDisplay` would be better?